Iterators and Closures/Standard Library Types/Intro/task.md

```diff
@@ -110,19 +111,19 @@ The following code snippet shows an example of calling the iterator adaptor meth
 The warning we get is this:
 
 ```text
-    warning: unused `std::iter::Map` which must be used: iterator adaptors are lazy
-    and do nothing unless consumed
-     --> src/main.rs:4:5
-      |
-    4 |     v1.iter().map(|x| x + 1);
-      |     ^^^^^^^^^^^^^^^^^^^^^^^^^
-      |
-      = note: #[warn(unused_must_use)] on by default
+warning: unused `Map` that must be used
+ --> src/main.rs:4:5
+  |
+4 |     v1.iter().map(|x| x + 1);
+  |     ^^^^^^^^^^^^^^^^^^^^^^^^^
+  |
+  = note: `#[warn(unused_must_use)]` on by default
+  = note: iterators are lazy and do nothing unless consumed
 ```
 
 The code in the last example doesn’t do anything; the closure we’ve specified never gets called. The warning reminds us why: iterator adaptors are lazy, and we need to consume the iterator here.
 
-To fix this and consume the iterator, we’ll use the `collect` method, which we used in **[Chapter 12](https://doc.rust-lang.org/stable/book/ch12-01-accepting-command-line-arguments.html)** with `env::args`. This method consumes the iterator and collects the resulting values into a collection data type.
+To fix this and consume the iterator, we’ll use the `collect` method, which is discussed in **[Chapter 12](https://doc.rust-lang.org/stable/book/ch12-01-accepting-command-line-arguments.html)** of the Rust Book with `env::args`. This method consumes the iterator and collects the resulting values into a collection data type.
```

Comment:
To fix this and consume the iterator, we’ll use the `collect` method, which is discussed in **[Chapter 12](https://doc.rust-lang.org/stable/book/ch12-01-accepting-command-line-arguments.html)** of the Rust Book, with `env::args`. This method consumes the iterator and collects the resulting values into a collection data type.